### PR TITLE
feature/pwa-config

### PR DESCRIPTION
**Pull request by Czaroos**
Configured app to be PWA-ready. At the moment default config is provided for caching.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,9 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="/public/favicon.ico" />
```
Comment (polubis) [inline public/index.html:5]: Just question. Why you removed this variable ?

File: webpack.config.js [hunk 2]
```diff
@@ -100,6 +101,16 @@ module.exports = (env, { mode }) => {
...
         //   ? "'https://pillar-api-dev.azurewebsites.net/api/'"
         //   : "'https://pillar-api.azurewebsites.net/api/'"
+      }),
+      new WorkboxPlugin.GenerateSW({
+        clientsClaim: true,
+        skipWaiting: true,
+        runtimeCaching: [
+          {
+            urlPattern: new RegExp('localhost:3000/*'),
+            handler: 'StaleWhileRevalidate'
```
Comment (polubis) [inline webpack.config.js:111]: Also can you add here some examples with caching API endpoint results ?

**Review by polubis — CHANGES_REQUESTED** [review]
Good job - some questions added

File: public/index.html (hunk 1 above)
Comment (Czaroos) [reply public/index.html]: for urls testing purposes and forgot to put it back later - changed to previous already

**Review by polubis — APPROVED** [review]

**Pull request merged**